src/rules.jl

```diff
@@ -213,10 +213,22 @@ struct GammaRule{T<:Real} <: AbstractLRPRule
     γ::T
     GammaRule(gamma=LRP_DEFAULT_GAMMA) = new{eltype(gamma)}(gamma)
 end
+is_compatible(rule::GammaRule, layer::Scale) = true
 function modify_parameters(r::GammaRule, param::AbstractArray)
     γ = convert(eltype(param), r.γ)
     return @. param + γ * keep_positive(param)
 end
+function modify_layer(rule::GammaRule, layer::Scale; keep_bias=true)
```

Comment:
Do we have to specify the `GammaRule` here or can this be a more generic function compatible with other rules?

Reply:
Hmm, I think you know this better than me. `layer = Scale(n)` is basically a simpler `Dense` layer, that computes `x -> layer.scale .* x .+ layer.bias`.

Reply:
So i guess `modify_weight` should work the same way on `layer.scale`; so it should also work for other rules.

Reply:
I'd remove the `GammaRule` restriction for now. Other weight modifications might be valid too.

Reply:
```suggestion
function modify_layer(rule, layer::Scale; keep_bias=true)
```